### ISSUE TITLE
Using string as easing function name in 'via' does not work

### ISSUE THREAD

**Issue by eysteinn** [mobius3/tweeny#27]
The code given as example in tween.h:

auto tween = tweeny::from(0.0f, 100.0f).to(100.0f, 0.0f).via(tweeny::easing::linear, "backOut");

results in a compilation error.

**Comment by mobius3** [comment]
Can you try 01d3e84477b9861c613346dabe804593c095e1c2?

**Comment by Arcnor** [comment]
@mobius3 I've had a similar issue today using this library (I'm not using string forms, but for some reason it started complaining about `string` later). Will it be possible to cut a new release with the latest commits, so package managers can update their versions?

Thanks in advance.

**Comment by mobius3** [comment]
I'm actually inclined to removing `via` with strings altogether. The use case was having it being set when those values comes from configuration files but it has caused more problems than solutions. Ideally this week I'll remove that and push a new release.

**Comment by Arcnor** [comment]
> I'm actually inclined to removing `via` with strings altogether. The use case was having it being set when those values comes from configuration files but it has caused more problems than solutions. Ideally this week I'll remove that and push a new release.

Yeah, whoever needs that can probably just do the switch themselves with the values they need anyway :).

Thanks!

**Comment by Arcnor** [comment]
> I'm actually inclined to removing `via` with strings altogether. The use case was having it being set when those values comes from configuration files but it has caused more problems than solutions. Ideally this week I'll remove that and push a new release.

Hey there @mobius3 , have you found the time to do this change? If you haven't, do you need help with it? Not sure if the fix is just remove the whole function altogether or something else, but I can open a PR if it helps. 

Thanks again!

**Comment by mobius3** [comment]
Huh, "this weekend", yeah. Sorry, life is taking all of me :P I'm willing to work together in a PR and accept it, indeed. The fix is just to nuke whatever string version of `via` there is, yes. Thanks for the help :)
